learning/tour-of-beam/backend/internal/storage/image/Dockerfile

```diff
@@ -24,7 +24,7 @@ FROM google/cloud-sdk:$GCLOUD_SDK_VERSION
 VOLUME /opt/data
 
 # RUN mkdir -p /opt/data/WEB-INF
-COPY index.yaml /opt/data/WEB-INF/index.yaml
+# COPY index.yaml /opt/data/WEB-INF/index.yaml
```

Comment:
Should we just remove this entirely? Why did we end up moving this?

Reply:
This was my initial idea of how to validate Datastore indexes: by copying it to the Datastore emulator container
But that actually doesn't work: emulator updates WEB-INF/index.yaml when it executes the query which requires a composite index. 
So, in this PR we switch to comparing the generated index.yaml with our VCS version.
Assuming integration tests covered every possible Datastore query, it produces an index that must work when deployed to Cloud

This line is removed